src/components/funnels/funnels/register/steps/ticket/level.tsx

```diff
@@ -42,7 +40,7 @@ const TicketLevel = (_: ReadonlyRouteComponentProps) => {
 							<Localized
 								key={id}
 								id={`register-ticket-level-card-${id}`}
-								attrs={{ label: true, priceLabel: true }}
+								attrs={{ label: true, priceLabel: true, priceLabelDay: true }}
```

Comment:
Could you please undo the priceLabelDay and isDayTicket properties here, it's better to do what I mention below in en.ftl